packages/types-dev/objects.d.ts

```diff
@@ -430,7 +430,7 @@ declare global {
 
         interface UserCommon extends ObjectCommon {
             /** The username */
-            name: string;
+            name: StringOrTranslated;
```

Comment:
I am still not a big fan of having translated usernames anywhere. If necessary - okay..

Reply:
It is already used. `administrator` group and `admin  user are already multi-language